server/controllers/Contacts.js

```diff
@@ -90,7 +90,7 @@ module.exports.getContacts = async (req, res) => {
 
 // Delete a Contact
 module.exports.deleteContact = async (req, res) => {
-    let userId = req.params.userId;
+    let userId = req.user.id;
     let contactId = req.body.contactId;
```

Comment:
Comment below can be removed because we no longer need to add that check since the user id comes from the jwt